packages/runtime/src/lib/constants.ts

```diff
@@ -167,6 +171,8 @@ export const ITxArrFields: {[key: number]: Set<string>} = {
   5: new Set(['Logs'])
 };
 
+ITxnFields[6] = ITxnFields[5];
```

Comment:
instead of assigning, we should make a copy using deepClone or `[...ITxnFields[5]]`

Reply:
I will use cloneDeep from lodash.